minhton/src/core/node.cpp

```diff
@@ -190,7 +190,7 @@ void MinhtonNode::prepareSending(const MessageVariant &msg_variant) {
           throw InvalidMessageException(header);
         }
 
-        FSMState state = FSMState(fsm_.current_state());
+        auto state = FSMState(fsm_.current_state());
```

Comment:
In my opinion, it would be better to use the syntax like in the line below to construct an object, what do you think?

Reply:
This cannot changed like the line below because ``FSMState`` is an enum. C++17 allows list initialization with scoped enums or unscoped enums when specifying the underlying type (https://en.cppreference.com/w/cpp/language/enum). 
I switched to this initialization with the last commit.